docs/registry/upload/_index.md

```diff
@@ -288,47 +290,120 @@ To update an existing module in the [Viam registry](https://app.viam.com/registr
    If you have not yet created any GitHub actions, click the **Set up a workflow yourself** link.
    See the [GitHub actions documentation](https://docs.github.com/en/actions/creating-actions) for more information.
 
-1. Paste the following action template YAML into the edit window.
-
-   ```yaml {class="line-numbers linkable-line-numbers"}
-   on:
-     push:
-     release:
-       types: [released]
-
-   jobs:
-     publish:
-       runs-on: ubuntu-latest
-       steps:
-         - uses: actions/checkout@v3
-         - name: build
-           run: echo "your build command goes here" && false # <-- replace this with the command that builds your module's tar.gz
-         - uses: viamrobotics/upload-module@v1
-           # if: github.event_name == 'release' # <-- once the action is working, uncomment this so you only upload on release
-           with:
-             module-path: module.tar.gz
-             platform: linux/amd64 # <-- replace with your target architecture, or your module will not deploy
-             version: ${{ github.event_name == 'release' && github.ref_name || format('0.0.0-{0}.{1}', github.ref_name, github.run_number) }} # <-- see 'Versioning' section below for explanation
-             key-id: ${{ secrets.viam_key_id }}
-             key-value: ${{ secrets.viam_key_value }}
-   ```
+1. Select the `build-action` or `upload-module` action and paste the following action template YAML into the edit window.
```

Comment:
Nit: `select` suggests a UI element in the GH UI that the user should find and interact with. We mean: pick the right Docs tab below. What do you think of:

```suggestion
1. Paste the following action template YAML into the edit window, depending on whether you are using the `build-action` or `upload-module` action:
```